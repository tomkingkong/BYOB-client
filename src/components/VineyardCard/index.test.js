import React from 'react';
import { shallow } from 'enzyme';
import { VineyardCard } from '.';

describe('VineyardCard', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<VineyardCard />);
    expect(wrapper).toMatchSnapshot();
  })
})