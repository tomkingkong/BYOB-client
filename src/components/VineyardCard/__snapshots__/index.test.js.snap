// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VineyardCard should match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <VineyardCard />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <article
          className="vineyard"
          style={
            Object {
              "display": "flex",
            }
          }
        >
          <div
            className="details"
          >
            <h2 />
            <p />
            <p />
            <button
              className="view-wines"
              onClick={[Function]}
            >
              View Wine List
            </button>
          </div>
        </article>,
        false,
        false,
        false,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="details"
          >
            <h2 />
            <p />
            <p />
            <button
              className="view-wines"
              onClick={[Function]}
            >
              View Wine List
            </button>
          </div>,
          "className": "vineyard",
          "style": Object {
            "display": "flex",
          },
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h2 />,
              <p />,
              <p />,
              <button
                className="view-wines"
                onClick={[Function]}
              >
                View Wine List
              </button>,
            ],
            "className": "details",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": undefined,
              },
              "ref": null,
              "rendered": null,
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": undefined,
              },
              "ref": null,
              "rendered": null,
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": undefined,
              },
              "ref": null,
              "rendered": null,
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "View Wine List",
                "className": "view-wines",
                "onClick": [Function],
              },
              "ref": null,
              "rendered": "View Wine List",
              "type": "button",
            },
          ],
          "type": "div",
        },
        "type": "article",
      },
      false,
      false,
      false,
    ],
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <article
            className="vineyard"
            style={
              Object {
                "display": "flex",
              }
            }
          >
            <div
              className="details"
            >
              <h2 />
              <p />
              <p />
              <button
                className="view-wines"
                onClick={[Function]}
              >
                View Wine List
              </button>
            </div>
          </article>,
          false,
          false,
          false,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="details"
            >
              <h2 />
              <p />
              <p />
              <button
                className="view-wines"
                onClick={[Function]}
              >
                View Wine List
              </button>
            </div>,
            "className": "vineyard",
            "style": Object {
              "display": "flex",
            },
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h2 />,
                <p />,
                <p />,
                <button
                  className="view-wines"
                  onClick={[Function]}
                >
                  View Wine List
                </button>,
              ],
              "className": "details",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": undefined,
                },
                "ref": null,
                "rendered": null,
                "type": "h2",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": undefined,
                },
                "ref": null,
                "rendered": null,
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": undefined,
                },
                "ref": null,
                "rendered": null,
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "View Wine List",
                  "className": "view-wines",
                  "onClick": [Function],
                },
                "ref": null,
                "rendered": "View Wine List",
                "type": "button",
              },
            ],
            "type": "div",
          },
          "type": "article",
        },
        false,
        false,
        false,
      ],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
